src/main/java/org/spin/pos/service/cash/CollectingManagement.java

```diff
@@ -168,6 +169,12 @@ public static  MPayment createPaymentFromOrder(MOrder salesOrder, CreatePaymentR
 		if(paymentMethodId > 0) {
 			payment.set_ValueOfColumn(I_C_PaymentMethod.COLUMNNAME_C_PaymentMethod_ID, paymentMethodId);
 		}
+		//	Allocate Payment Method
+		if(allocatePaymentId > 0) {
+			PO paymentTypeAllocation = POS.getPaymentTypeAllocationId(allocatePaymentId, null);
```

Comment:
please validate if `paymentTypeAllocation` variable is null or `paymentTypeAllocation.get_ID()` is >=  zero, to set isOnline

Reply:
[f15893e](https://github.com/solop-develop/adempiere-grpc-server/pull/1120/commits/f15893e3fce9421bbe76e139035adac0ab23944c)